pkg/config/configsections/Node.go

```diff
@@ -0,0 +1,43 @@
+// Copyright (C) 2020-2021 Red Hat, Inc.
+//
+// This program is free software; you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation; either version 2 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License along
+// with this program; if not, write to the Free Software Foundation, Inc.,
+// 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+
+package configsections
+
+// Nodes defines in the cluster. with name of the node and the type of this node master/worker,,,,,
+type Node struct {
+	Name string
+	Type []string
+}
+
+var worker = "worker"
+var master = "master"
+
+func (Node) IsMaster(node Node) bool {
+	for i := range node.Type {
+		if node.Type[i] == master {
+			return true
+		}
+	}
+	return false
+}
+func (Node) IsWorker(node Node) bool {
```

Comment:
the function should not have node as parameter. 
it should have a signature like ``func (n Node) isWorker() bool{}``